pkg/controllers/vdb/sandboxsubcluster_reconciler.go

```diff
@@ -378,7 +382,8 @@ func (s *SandboxSubclusterReconciler) sandboxSubcluster(ctx context.Context, sub
 	return ctrl.Result{}, nil
 }
 
-// updateSandboxStatus will update sandbox status in vdb
+// updateSandboxStatus will update sandbox status in vdb. This is a bulk update
+// and can handle multiple subclusters at once.
 func (s *SandboxSubclusterReconciler) updateSandboxStatus(ctx context.Context, originalSbScMap map[string][]string) error {
```

Comment:
Looks like this is no longer needed.

Reply:
I see this is still used in `reconcileSandboxStatus()`

Reply:
Yes, we still need the bulk update.